.github/PULL_REQUEST_TEMPLATE/refactor_template.md

```
@@ -0,0 +1,15 @@
+### PR 타입
+
+REFACTOR : 코드 리펙토링
+
+### 반영 브랜치
+
+ex) feat/login -> dev
+
+### 변경 사항
+
+#이슈번호 or JIRA ID
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)